test/Unosquare.Swan.Test/CsvWriterTest.cs

```diff
@@ -18,80 +18,91 @@ public class CsvWriterTest
 Co,2,""C#, MySQL, JavaScript, HTML5 and CSS3"","" $1,359,885 "" 
 Ca,2,""C#, MySQL, JavaScript, HTML5 and CSS3"","" $1,359,885 """;
 
-        [Test]
-        public void ConstructorTest()
+        [TestFixture]
```

Comment:
Let's use a different namespace for every tested class..namespace Unosquare.Swan.Test.CsvWriter. Would be easier to read the test report